flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+  "publish-delay-hours": 1,
```

Comment:
Not sure what snapcraft promises in terms of availability of URLs but it's probably safer to set it to 0 to make it published immediately after flatpak-external-data-checker merges an update.